packages/material-ui-lab/src/useAutocomplete/useAutocomplete.js

```diff
@@ -680,7 +673,7 @@ export default function useAutocomplete(props) {
       onChange: handleInputChange,
       // if open then this is handled imperativeley so don't let react override
       // only have an opinion about this when closed
-      'aria-activedescendant': popupOpen ? undefined : null,
+      'aria-activedescendant': popupOpen ? '' : null,
```

Comment:
I haven't looked in detail how React handles undefined vs null. One key element is that, as far as I know, React doesn't diff with the DOM, but with the preview tree representation, he has. I suspect treats both values the same way, hence doesn't apply a diff. In any case, our test cases are our belt and straps.